scripts/package/macos-notarize-app.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+if [ -z "$1" ]; then
+    echo "Specify app bundle as first parameter"
+    exit 1
+fi
+
+if [ -z "$APPLE_ID_USER" ] || [ -z "$APPLE_ID_PASSWORD" ]; then
+    echo "You need to set your Apple ID credentials with \$APPLE_ID_USER and \$APPLE_ID_PASSWORD."
+    exit 1
+fi
+
+APP_BUNDLE=$(basename "$1")
+APP_BUNDLE_DIR=$(dirname "$1")
+
+cd "$APP_BUNDLE_DIR"
+
+# Package app for submission
+echo "Generating ZIP archive ${APP_BUNDLE}.zip..."
+ditto -c -k --rsrc --keepParent "$APP_BUNDLE" "${APP_BUNDLE}.zip"
+
+# Submit for notarization
+echo "Submitting $APP_BUNDLE for notarization..."
+RESULT=$(xcrun altool --notarize-app --type osx \
+  --file "${APP_BUNDLE}.zip" \
+  --primary-bundle-id org.musicbrainz.Picard \
+  --username $APPLE_ID_USER \
+  --password @env:APPLE_ID_PASSWORD \
+  -itc_provider MetaBrainzFoundationInc \
```

Comment:
Other options are using double dash, but not this one, expected?

Reply:
Yes, that's how this is spelled. Don't ask why. Unlike the other options this option also doesn't show up if you run `xcrun altool --help` 🤷‍♂️

I had to Google a bit to find it. The reason why we need it is that my Apple Developer account is associated with more than one subscription. This parameter selects the proper subscription.